src/client/src/components/RightSidebar/index.tsx

```diff
@@ -95,13 +115,22 @@ const RightSidebar = (props: Props) => {
               {selectedRoute !== ROUTE.REVIEW_AND_GENERATE && (
                 <div className={styles.buttonContainer}>
                   <button
-                    className={classnames(buttonStyles.buttonDark, styles.button)}
+                    className={buttonStyles.buttonLink}
                     onClick={() => dispatch(ModalActions.openViewLicensesModalAction())}
                   >
                     {formatMessage(messages.viewLicenses)}
                   </button>
                 </div>
               )}
+              {hasRequirements && (
+                <button
+                  className={buttonStyles.buttonLink}
+                  onClick={() => dispatch(ModalActions.openViewPlatformRequirementsAction())}
```

Comment:
same here remove "View" if we rename the previous one.

Reply:
Done.